src/styles.css

```diff
@@ -44,134 +26,10 @@ a {
 a:hover {
   color: #006080;
 }
```

Comment:
these can further delete or can't liao?

Reply:
can't liao if not all the a tag will need to have the hover.